CHANGES.md

```diff
@@ -74,6 +75,268 @@
 * [Version 1.0.1](#version-101)
 * [Version 1.0](#version-10)
 
+## Version 3.3.0 (September 30, 2024)
+
+Minor release two significant changes:
+
+- The C++ API now uses the OpenEXRCore library underneath.
+
+- New API for accessing compression types
+
+  In anticipation of future support for new compression types, there
+  is now a convenience API for mapping between compression type names
+  and the associated enum:
+  
+  - ``getCompressionDescriptionFromId(Compression, std::string&)``
+  - ``getCompressionIdFromName(const std::string&, Compression&)``
+  - ``getCompressionNameFromId(Compression, std::string&)``
+  - ``getCompressionNamesString(const std::string&, std::string&)``
+  - ``getCompressionNumScanlines(Compression)``
+  - ``isValidCompression(int)``
+
+- New bin tools:
+
+  - ``exrmetrics`` - Read an OpenEXR image from infile, write an
+    identical copy to outfile reporting time taken to read/write and
+    file sizes.
+
+  - ``exrmanifest`` - Read exr files and print the contents of the
+    embedded manifest.
```

Comment:
"manifest" is going to be new for some people, wonder if we should have a note for learning more about it?

Reply:
something like "... of the embedded manifest.  The manifest provides a mapping to a human-readable identifiers, see https://github.com/AcademySoftwareFoundation/openexr/blob/25c2d775f889948d48114f382b58e35d313ecad3/website/DeepIDsSpecification.rst for more information."